src/test/scala/org/apache/spark/sql/connector/kinesis/retrieval/client/FakePollingClientConsumerFactory.scala

```diff
@@ -232,10 +232,10 @@ object FakePollingClientConsumerFactory {
   }
 
 
-  private class SingleShardEmittingFixNumOfRecordsWithExpiredIteratorKinesis(numOfRecords: Int,
-                                                                             numOfGetRecordsCalls: Int,
-                                                                             orderOfCallToExpire: Int,
-                                                                             millisBehindLatest: Long)
+  class SingleShardEmittingFixNumOfRecordsWithExpiredIteratorKinesis(numOfRecords: Int,
```

Comment:
Was this required by one of the upgrades? If yes, which one?

Reply:
It was required by Mockito upgrade. I don't have exact error message now, it was related to mockito not being able to access private class.